### PR TITLE
Add option to close dropdown on select. Closes #106

### LINKED ISSUE #106: Close Multi select after selection made
The control should close the dropdown after a selection is made.  Regardless if it's multiple or not.

### PR DESCRIPTION
**Pull request by jarib**
New option `closeDropdownOnSelect` should solve https://github.com/mskocik/svelecte/issues/106.

### CODE REVIEW

File: src/settings.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const settings = {
   resetOnBlur: true,
   resetOnSelect: true,
   fetchResetOnBlur: true,
+  closeDropdownOnSelect: true,
```
Comment (mskocik) [inline src/settings.js]: Can you just set default value to `false` to keep current behavior?

**Review by mskocik — CHANGES_REQUESTED** [review]

File: src/settings.js (hunk 1 above)
Comment (jarib) [reply src/settings.js]: Done!

**Comment by jarib** [comment]
I am btw not completely sure about the naming here, since `multiple={false} closeDropdownOnSelect={false}` would still close the dropdown with this implementation.

**Review by mskocik — APPROVED** [review]

**Comment by mskocik** [comment]
Yes, but it's fine, because this option has meaning for multiselect only anyway

**Comment by mskocik** [comment]
I've renamed the prop to make it shorter and added missing documentation, which I forgot to check in the PR 😆

**Comment by jarib** [comment]
Perfect! Thank you.

**Pull request merged**